### PR TITLE
#29-custompattern

### PR DESCRIPTION
**Pull request by AkshayGadikar**
Custom Pattern

### CODE REVIEW

File: internal/pattern/DefaultHttpPattern.json [hunk 1]
```diff
@@ -20,10 +20,6 @@
         "sub": "=$.conf.jwtSub"
       }
     },
-    {
```
Comment (pointlander) [inline internal/pattern/DefaultHttpPattern.json]: This call to the circuit breaker is needed. Why did you remove it?

Comment (AkshayGadikar) [reply internal/pattern/DefaultHttpPattern.json]: Fixed
